api/app/responses/api.py

```diff
@@ -106,6 +106,12 @@ def post(self):
                 user = user_repository.get_by_id(user_id)
                 response = response_repository.get_by_id_and_user_id(response.id, user_id)
                 self.send_confirmation(user, response)
+                
+                event = event_repository.get_event_by_response_id(response.id)
+                if event.event_type == EventType.CONTINUOUS_JOURNAL:
+                    all_admin_users = event_repository.get_all_admin(event_id=response.id)
```

Comment:
I think there's a typo here setting the `event_id` and I think also let's rather rename the variable to `all_admin_event_roles` to match the type we're working with. I know it's a little awkward since the class `EventRole` is actually an `EventUserRole` but I think it still helps with readability (especially since we do have an `AppUser` class which we generally name `users` so we need to make it clear they're different).
```python
all_admin_event_roles = event_repository.get_all_admin(event_id=event.id)
```